tests/perf-system/submitter/handle_arguments.h

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+
+#pragma once
+
+#include "ccf/ds/logger.h"
+
+#include <CLI11/CLI11.hpp>
+#include <iostream>
+
+class ArgumentParser
+{
+public:
+  std::string label; //< Default set in constructor
+  std::string pid_file; //< Default set in constructor
+
+  std::string cert;
+  std::string key;
+  std::string rootCa;
+  std::string server_address = "127.0.0.1:8000";
+  std::string send_filepath = "../tests/perf-system/submitter/cpp_send.parquet";
+  std::string response_filepath =
+    "../tests/perf-system/submitter/cpp_respond.parquet";
+  std::string generator_filepath =
+    "../tests/perf-system/generator/requests.parquet";
+  bool pipeline = false;
+
+  ArgumentParser(
+    const std::string& default_label,
+    const std::string& default_pid_file,
+    CLI::App& app) :
+    label(default_label),
+    pid_file(fmt::format("{}.pid", default_pid_file))

```

Comment:
What do we use the pid file for? That's generally useful for a long running server/daemon we detach, but I don't expect we will ever use the submitter like that.